### DISCUSSION TITLE
Range slider styling

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#273] [1. Help]
Hi @vovayatsyuk 

I am trying to re-style the range slider, but CSS is not being applied on that, I also checked the docs links https://breezefront.com/docs/range-slider, and tried to override the CSS variable but could not.

Can you please help on this ?

**Comment by vovayatsyuk** [comment]
I recommend using browser dev tools to see what is wrong. Check if your styles are added to the page, then check if their priority is more than priority of the default styles.
